tool/internal/instrument/instrument.go

```diff
@@ -71,14 +79,12 @@ func (ip *InstrumentPhase) instrument(rset *rule.InstRuleSet) error {
 		}
 		// Since trampoline-jump-if is performance-critical, perform AST level
 		// optimization for them before writing to file
-		err = ip.optimizeTJumps()
-		if err != nil {
+		if err = ip.optimizeTJumps(); err != nil {
```

Comment:
I don't have a strong preference, but given that the project currently uses the ["happy path"](https://go.dev/wiki/CodeReviewComments#indent-error-flow) idiom, I'd suggest we follow that here for consistency. (The alternative would be to have a single patch dedicated to changing the style, but we should avoid mixing styles)

Reply:
> Try to keep the normal code path at a minimal indentation
> If the if statement has an initialization statement, such as:
> 
> if x, err := f(); err != nil {
>     // error handling
>     return
> } else {
>     // use x
> }
> then this may require moving the short variable declaration to its own line:
> 
> x, err := f()
> if err != nil {
>     // error handling
>     return
> }
> // use x